### DISCUSSION TITLE
Error during Request a file in transfer/transfer-01-file-transfer

### DISCUSSION THREAD

**Discussion by AbinayaThulsi** [eclipse-edc/Samples#29] [Q&A]
After initiating the transferprocess command : curl -X POST -H "Content-Type: application/json" -H "X-Api-Key: password" -d @transfer/transfer-01-file-transfer/filetransfer.json "http://localhost:9192/api/v1/management/transferprocess"

i get the below error:

TransferProcess e68cadbb-8bca-40c3-ae7d-72e4b5912ee6 stays in state REQUESTING.
java.lang.IllegalArgumentException: Illegal character in opaque part at index 22: urn:contractagreement:{agreement ID}

I have updated the contact agreement id and distination path in filetransfer.json file. But with or without update the below error occurs. Could some help?

Thank you 

<img width="468" alt="image" src="https://user-images.githubusercontent.com/28295277/222553349-e0430430-2c28-4b23-aa55-8a95d3056894.PNG">

**Comment by ndr-brt** [comment]
This happened because you didn't updated the `filetransfer.json` as specified in the [README at point 4](https://github.com/eclipse-edc/Samples/tree/main/transfer/transfer-01-file-transfer#4-request-the-file)

**Reply by AbinayaThulsi** [reply]
Hi, 

Thank you for the reply. As mentioned in my post earlier, I updated both the agreement id and data destination path in the filetransfer.json file, even then the error occurred.

Solution/Reason for getting the error: I was using ubuntu terminal and later when I tried the same in command prompt I did not receive the above error. It could probably be the differences in the configuration of the two command line interfaces.

kind regards
